torch/_refs/__init__.py

```diff
@@ -366,7 +368,7 @@ def __maybe_broadcast(x, shape):
 
 
 # Utilities should come BEFORE this import
-from torch._decomp import register_decomposition
+from torch._decomp import register_decomposition, shape_preserving_default
```

Comment:
@Chillee prefers this entire logical set of things live in fake_tensor, I am not opposed, but open to other designs as well. Interested in feedback.